test.js

```diff
@@ -20,6 +20,13 @@ test('finds from options.cwd', t => {
 	t.is(findCacheDir({cwd: __dirname, name: 'bar'}), path.join(__dirname, 'node_modules', '.cache', 'bar'));
 });
 
+test('supports CACHE_DIR environmental variable', t => {
+	const newCacheDir = path.join(__dirname, 'new-cache-dir');
```

Comment:
Use https://github.com/sindresorhus/tempy to put it in a temporary location.

Reply:
Done!